tests/unit/ui/components/TestDropdownMenu.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import DropdownMenu, { DropdownMenuBase } from 'ui/components/DropdownMenu';
+import DropdownMenuItem from 'ui/components/DropdownMenuItem';
+import Icon from 'ui/components/Icon';
+import { createFakeEvent, shallowUntilTarget } from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  it('renders a menu', () => {
+    const menu = shallowUntilTarget(
+      <DropdownMenu text="Menu" />,
+      DropdownMenuBase
```

Comment:
since this is repeated for every call, you should definitely add a helper at the top of the test suite.